mall.py

```diff
@@ -108,15 +108,15 @@ def get_paged_videos(self, page, video_type):
         result = []
 
         if video_type == 'recent':
-            page = self.get_page('/sekce/nejnovejsi?page=%s' % page)
+            page = self.get_page('/sekce/nejnovejsi?page={0}'.format(page))
         elif video_type == 'popular':
-            page = self.get_page('/sekce/nejsledovanejsi?page=%s' % page)
+            page = self.get_page('/sekce/nejsledovanejsi?page={0}'.format(page))
 
         videos = self.extract_videos(page, search_section=(page == 0))
 
         for r in videos:
             ctx_url = self.url_for('show', link=r['show_link'])
-            r['label'] = '[LIGHT]%s[/LIGHT] | %s' % (r['show_name'], r['label'])
+            r['label'] = '[LIGHT]{0}[/LIGHT] | {1}'.formats(r['show_name'], r['label'])
```

Comment:
preklep - formats neexistuje. 
+ skonci na to unicode vyjimce, vrat tam prosim puvodni verzi.